packages/interactivity/src/directives.tsx

```diff
@@ -50,6 +50,39 @@ function deepClone< T >( source: T ): T {
 	return source;
 }
 
+function wrapEventAsync( event: Event ) {
+	const handler = {
+		get( target: any, prop: any, receiver: any ) {
```

Comment:
```suggestion
		get( target: Event, prop: string, receiver: any ) {
```

Maybe the typing of `receiver` could be made more specific as well.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/68097/commits/682ee6f3fb8647e664287df73c77ce79d6cc3e1a

I used `string | symbol` for `prop`, in accordance with other proxy handlers already present in Interactivity API code. Those handlers also use `receiver: any`, so I think it's fine to leave that as-is.